pkg/kritis/review/review.go

```diff
@@ -50,42 +50,113 @@ func New(c *Config) Reviewer {
 	}
 }
 
+// Flatten allowlist from a list of generic attestation policies
```

Comment:
I think I've reviewed all the individual pull requests that add up to this.  Does this file contain any changes I should pay special attention to?

Reply:
No. All changes are are new in this PR so far are due to merge fixes, IIRC.